components/feature/media/src/main/java/mozilla/components/feature/media/service/MediaServiceDelegate.kt

```diff
@@ -125,6 +126,9 @@ internal class MediaServiceDelegate(
                 .putString(
                     MediaMetadataCompat.METADATA_KEY_TITLE,
                     state.getActiveMediaTab().getTitleOrUrl(context))
+                .putString(
+                    MediaMetadataCompat.METADATA_KEY_ARTIST,
+                    state.getActiveMediaTab().nonPrivateUrl)
```

Comment:
`getTitleOrUrl(context))` is not an option here?

Reply:
No, then on the lock screen it will have the same line twice.  The title above is already using `getTitleOrUrl`.  The "artist" part is the same as the subtitle of the media notification.